okd/scripts/provision.sh

```diff
@@ -64,6 +71,7 @@ echo "nameserver 127.0.0.1" > /etc/resolv.conf
 
 mkdir -p /etc/dnsmasq.d
 echo "server=/tt.testing/192.168.126.1" >> /etc/dnsmasq.d/openshift.conf
+echo "address=/apps.tt.testing/192.168.126.51" >> /etc/dnsmasq.d/openshift.conf
```

Comment:
It works only on the provision stage and will not apply to the running stage, so I do not sure you need it here.
Also under the running phase, I am using HAProxy and do not run dnsmasq at all, because I had troubles to forward requests to the API server with the dnsmasq.

Reply:
i need to investigate this further. I'm not 100% sure if this is necessary.